src/tasks/initiate_new_auction.ts

```diff
@@ -8,17 +8,20 @@ import { getEasyAuctionContract } from "./utils";
 const initiateAuction: () => void = () => {
   task("initiateAuction", "Starts a new auction")
```

Comment:
Unrelated to this PR, but having a task to start an auction makes a lot of sense!